ansible_collections/arista/avd/roles/eos_designs/docs/tables/management-settings.md

```diff
@@ -107,6 +158,9 @@
           unit: <str>
 
         # Regular expression to use for searching log messages. Required for on-logging trigger.
+        # This key is deprecated.
+        # Support will be removed in AVD version 5.0.0.
+        # Use <samp>event_handlers.trigger_on_logging.regex</samp> instead.
```

Comment:
Make sure to move any tests with this to the `deprecated_vars` molecule (and make sure to add a test if we are missing it :) )